app/models/manageiq/providers/openshift/container_manager/refresher.rb

```diff
@@ -25,16 +25,19 @@ def fetch_hawk_inv(ems)
       end
 
       def parse_legacy_inventory(ems)
+        request_entities = OPENSHIFT_ENTITIES.dup
+        request_entities.delete(:name => 'images') if refresher_options.skip_container_images
```

Comment:
Consider turning OPENSHIFT_ENTITIES into a method.
I don't like there being a const that's not always the correct value.  Though now that you flipped it to include images unless skip, it's pretty OK as is.